Gems/Atom/RPI/Code/Tests/Material/MaterialTypeAssetTests.cpp

```diff
@@ -92,7 +93,29 @@ namespace UnitTest
         Ptr<ShaderOptionGroupLayout> m_testShaderOptionsLayout;
         Data::Asset<ShaderAsset> m_testShaderAsset;
         Data::Asset<ImageAsset> m_testImageAsset;
+        Data::Asset<ImageAsset> m_testImageAsset2;
         const char* m_testImageFilename = "test.streamingimage";
+        const char* m_testImageFilename2 = "test.streamingimage2";
```

Comment:
This is a little nitpicky but it would be better named "test2.streamingimage" instead of introducing a new extension.